README.md

```diff
@@ -14,13 +14,31 @@ Kibana is an open source ([Apache Licensed](https://github.com/elastic/kibana/bl
 * Run `bin/kibana` on unix, or `bin\kibana.bat` on Windows.
 * Visit [http://localhost:5601](http://localhost:5601)
 
-
 ## Upgrade from previous version
 
 * Move any custom configurations in your old kibana.yml to your new one
 * Reinstall plugins
 * Start or restart Kibana
 
+## Version compatibility with Elasticsearch
+
+Ideally, your Elasticsearch and Kibana version numbers should both be the same. If they're not, then you may run into a version incompatibility.
+
+In the table below, 1 represents a current version number, -1 represents an older version number, and 0 represents a version number that's irrelevant to the particular case.
+
+🚫 **Fatal errors** are caused by a) any differences in major version and b) Elasticsearch having an older version number than Kibana.
+
+⚠️ **Server warnings** are displayed in the terminal if Elasticsearch has a newer minor or patch version number than Kibana. Kibana will still run, but only to make it easier for you to upgrade it to match the Elasticsearch version. You should try to run the same versions of Kibana and Elasticsearch at all times.
+
+💚 **Everything is awesome** when you're part of a team. And when you're running the same versions of Kibana and Elasticsearch.
+
+| Kibana version | ES -1.0.0 | ES 1.-1.0 | ES 1.1.-1 | ES 1.1.1 |
```

Comment:
👍 to an English description first, but what would a `+1` in a column represent? The problem I kept running into when trying to figure out how to represent these numbers is that it's all relative. The only absolute value is a `1` which means "current version". Then we need some way to indicate versions which are old, which is why I went with `-1`.